### PR TITLE
Docs add plugins

### PR DESCRIPTION
**Pull request by predragnikolic**
These plugins are available on PC but were not documented.

### CODE REVIEW

**Comment by predragnikolic** [comment]
Because I updated only the documentation, Github Actions will not be run. (so ignore the waiting status)

File: docs/src/language_servers.md [hunk 1]
```diff
@@ -290,8 +294,16 @@ Follow installation instructions on [LSP-json](https://github.com/sublimelsp/LSP
 
 ## LaTeX
 
+There are multiple options:
+
+### TexLab
+
 Follow installation instructions on [LSP-TexLab](https://github.com/sublimelsp/LSP-TexLab).
 
+### LTeX
```
Comment (rwols) [inline docs/src/language_servers.md:303]: "LTeX" is not a language, no? We seem to be ordering these sections on language, not language server?

**Review by rwols — CHANGES_REQUESTED** [review]
Serenata doesn't work for ST4. So you should revert 8483633.

See https://github.com/sublimelsp/LSP/pull/1792

File: docs/src/language_servers.md (hunk 1 above)
Comment (predragnikolic) [reply docs/src/language_servers.md:303]: "LTeX" is the name of the language server.

When there is "There are multiple options" I put the name of the language server for each option.

Comment (rwols) [reply docs/src/language_servers.md:303]: But the other sections are (mainly) languages. The exception seems to be ESLint. LTeX-LS serves latex and markdown.

Comment (jwortmann) [reply docs/src/language_servers.md:303]: It's still under the LaTeX language section. This heading with the language server name is one level smaller, similar to the language server sections under Python or Ruby for example.

**Pull request merged**